src/scripts/cfo.zig

```diff
@@ -73,6 +73,48 @@ const Fuzzer = enum {
     vsr_journal_format,
     vsr_superblock_quorums,
     vsr_superblock,
+    vopr,
+    vopr_testing,
+
+    fn print_command(fuzzer: Fuzzer, shell: *Shell, seed: u64) ![]const u8 {
+        if (fuzzer == .vopr or fuzzer == .vopr_testing) {
+            const state_machine: []const u8 =
+                if (fuzzer == .vopr) "" else "-Dsimulator-state-machine=testing";
+            return try shell.print(
+                "./zig/zig build -Drelease{s} simulator_run -- {d}",
```

Comment:
```suggestion
                "./zig/zig build -Drelease {s} simulator_run -- {d}",
```

Reply:
Ah, good catch! This is intentionally without a space, but I am missing one above in `-Dsimulator-state-machine`!